python/whylogs/core/constraints/helpers.py

```diff
@@ -0,0 +1,252 @@
+from typing import Union
```

Comment:
I wonder if it would be better to have more namespaces to help distinguish between the constraints?
In the future, we'll also have Dataset level constraints and other types. It will be confusing to have them all under `helpers` (or other name, if we're going to change it)

Reply:
But even the currently ones could be distinguished by the metric it uses.

Reply:
what is your suggestion? is 
```
factories/metric_constraints.py 
factories/dataset_constraints.py
...
```
enough?

Reply:
firstly i had created different namespaces for each metric, but i ended up having to repeat all the imports in the modules and fixtures to create the very similar tests.

Reply:
Yes, I don't have a clear suggestion yet.
Maybe separating by each metric is too much granularity?

Right now, some come to mind:
- metric_constraints.py
   - multiple_metrics_constraint (this could live with the metrics_constraints, I guess)
- multiple_columns_constraints.py
- multiple_profile_constraints (some constraints need a baseline, like constraints for ks_test and chi_squared. whether this should be, in fact, a constraint, is also a relevant discussion)
- dataset_constraints.py

But since we have only metric_constraints, we could create a namespace for it and make the other ones as we implement them?

Reply:
yes, and they should all live either on 
whylogs.core.constraints (the actual classes that define them)
or
whylogs.core.constraints.factories (the factories with helper functions)

Reply:
I think they should live under `factories` and leave `core.constraints` for the "building blocks".